.github/workflows/ci.yml

```diff
@@ -0,0 +1,77 @@
+name: CI - Ansible Collection Quality Checks
+
+on:
+  pull_request:
+    branches:
+      - dev
+  push:
+    branches:
+      - dev
+
+jobs:
+  # Sanity tests
+  sanity-tests:
```

Comment:
Can we reference the sanity section in https://github.com/ansible-collections/collection_template/blob/main/.github/workflows/ansible-test.yml